README.md

```diff
@@ -1,59 +1,49 @@
+![image](https://user-images.githubusercontent.com/24784631/216797451-f3d58b02-c900-4d37-9a3c-03ce48ffc411.png)
```

Comment:
I wouldn't mind not having the logo, not adding much :D

Reply:
Unless you are very attached to it, would you mind dropping the image from the change :)? Other than that looks good.

Reply:
Dropped the image and changed it to a simple “LegionCore” header.